prepare-chroot-base

```diff
@@ -13,16 +13,6 @@ PLUGIN_DIR=$(dirname "$0")
 INSTALLDIR=$1
 DIST=$2
 
-RPM_VERSION="$(rpm --version | awk '{print $3}')"
-
-if [ "$(printf '%s\n' "$RPM_VERSION" "4.14.0" | sort -V | head -n1)" = "4.14.0" ]; then
-    PGP_INVALID='SIGNATURES NOT OK'
-    PGP_SIGNED='signatures OK'
-else
-    PGP_INVALID='PGP'
-    PGP_SIGNED=' pgp '
-fi
```

Comment:
If you remove support for older rpm (which I think is ok), please add explicit error message if rpm is too old. Otherwise user will get a confusing message about not signed package (when in fact it was signed).